src/status/functions/sendRawStatus.ts

```diff
@@ -132,4 +134,10 @@ webpack.onInjected(() => {
 
     return await func(...args);
   });
+
+  wrapModuleFunction(GROUP_JID, async (func, ...args) => {
+    if (args[0].toString().includes('broadcast')) {
+      return await CHAT_JID(...args);
+    }
```

Comment:
Faltou apenas chamar a função original para outros casos.
```suggestion
    }
    return await func(...args);
```